src/dialogs/drives.py

```diff
@@ -78,7 +78,7 @@ def __remove(self, widget):
 
 
 @Gtk.Template(resource_path='/com/usebottles/bottles/dialog-drives.ui')
-class DrivesDialog(Handy.Window):
+class DrivesDialog(Adw.Window):
```

Comment:
ideally we wouldn't have this and other as AdwWindow's - they were only Hdy* for rounded corners, which is unneccesary now